class.jetpack-modules-list-table.php

```diff
@@ -294,12 +301,35 @@ function column_name( $item ) {
 
 	function column_description( $item ) {
 		ob_start();
+		/**
+		 * Filters each module's short description.
+		 *
+		 * @since 3.0.0
+		 *
+		 * @param string $item['description'] Module description.
+		 * @param string $item['module'] Module name.
```

Comment:
`$item['module']` is the module slug in this case, which differs from the `name` we use in js templates that is pulled directly from the module header.  

Also, I think `Jetpack_Admin::get_modules();` might be a better place to document these.  I can't find any evidence that we use this method anymore.